apps/admin/frontend/src/screens/write_ins_adjudication_screen.tsx

```diff
@@ -117,6 +117,7 @@ const BallotViews = styled.div`
 const AdjudicationControls = styled.div`
   display: flex;
   flex-direction: column;
+  width: 575px;
```

Comment:
I wonder if this should be set as a proportion of the screen width in case we ever change screen sizes? And/or whether it should be in `rem` rather than pixels for consistency with the rest of our styles?